From 0673033d71f4290662206feb75398f12cfd834e2 Mon Sep 17 00:00:00 2001
From: Suneel Garapati <sgarapati@caviumnetworks.com>
Date: Fri, 11 Oct 2019 13:35:09 -0700
Subject: [PATCH 0968/1239] board: octeontx2: fix incorrect mac_addr initialize

Board file cleanup slipped in incorrect init of mac
address in octeontx2_board_get_mac_addr, so fix it.

Change-Id: I78c893380547b497c84152c1a0c3cad0e2fc55fa
Signed-off-by: Suneel Garapati <sgarapati@caviumnetworks.com>
Reviewed-on: https://sj1git1.cavium.com/17246
---
 board/Marvell/octeontx2/board.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/board/Marvell/octeontx2/board.c b/board/Marvell/octeontx2/board.c
index 89d58deeb9..f199549630 100644
--- a/board/Marvell/octeontx2/board.c
+++ b/board/Marvell/octeontx2/board.c
@@ -43,6 +43,7 @@ void octeontx2_board_get_mac_addr(u8 index, u8 *mac_addr)
 
 	board_mac_num = fdt_get_board_mac_cnt();
 	if ((!is_zero_ethaddr((u8 *)&board_mac_addr)) && board_mac_num) {
+		tmp_mac = board_mac_addr;
 		tmp_mac += index;
 		tmp_mac = swab64(tmp_mac) >> 16;
 		memcpy(mac_addr, (u8 *)&tmp_mac, ARP_HLEN);
-- 
2.29.0

